Playbooks/AS-Slack-Integration/README.md

```diff
@@ -0,0 +1,168 @@
+![Slack_App_1](Images/Slack_App_1.png)
+
+
+Enter in "**Microsoft Sentinel**" for the app name then add a description. 
+Finally, select a workspace and click "**Create App**" .
+
+![Slack_App_2](Images/Slack_App_2.png)
+
+
+Now we need to add permissions to the app in order for it to post messages in channels. 
+From the "**Basic Information**" section of your newly created app, click "**Permissions**".
+
+![Slack_App_3](Images/Slack_App_3.png)
+
+
+Scroll down to the "**Scopes**" section and click the "**Add an OAuth Scope**" button under "**Bot Token Scopes**".
+Select the "**chat:write**" option.
+
+![Slack_App_4](Images/Slack_App_4.png)
+
+
+Scroll back up to the top of the page and click the "**Install to Workspace**" button, which should now be green.
+
+![Slack_App_5](Images/Slack_App_5.png)
+
+
+Click "**Allow**" and then copy the entire value in the "**Bot User OAuth Token**" field. 
+
+![Slack_App_6](Images/Slack_App_6.png)
+
+
+If you would like, you can also add the Microsoft Sentinel Logo under "**Display Information**".
+
+![Slack_App_7](Images/Slack_App_7.png)
+
+
+Your Slack app is now configured. However, before it can post messages, it needs to be invited to the desired channel.
+Type "**/invite**" in the desired channel, then select the "**Add apps to this channel**" option.
+
+![Slack_App_8](Images/Slack_App_8.png)
+
+
+Find your Microsoft Sentinel app and click "**Add**".
+
+![Slack_App_9](Images/Slack_App_9.png)
+
+
+Lastly, you will need to take note of your channel id for the playbook deployment.
+This can be found by running the browser version of slack and looking at the url of the desired channel, like so:
+
+https://app.slack.com/client/{workspace-id}/{**channel-id**}}
+
+
+#
+### Deployment   
+
+To configure and deploy this playbook:
+
+<a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2FAzure-Sentinel%2Fmaster%2FPlaybooks%2FAS-Slack-Integration%2Fmain%2Fazuredeploy.json" target="_blank">
+    <img src="https://aka.ms/deploytoazurebutton""/>
+</a>
+<a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2FAzure-Sentinel%2Fmaster%2FPlaybooks%2FAS-Slack-Integrationn%2Fmain%2Fazuredeploy.json" target="_blank">
+<img src="https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/1-CONTRIBUTION-GUIDE/images/deploytoazuregov.png"/>
+</a>                                                 
+
+Click the “**Deploy to Azure**” button at the bottom and it will bring you to the custom deployment template.
+
+In the **Project Details** section:
+
+* Select the “**Subscription**” and “**Resource Group**” from the dropdown boxes you would like the playbook deployed to.  
+
+In the **Instance Details** section:   
+
+* **Playbook Name**: This can be left as “AS-Slack-Integration” or you may change it.  
+
+* **OAuth Token**: Enter the value of the Slack OAuth Token created from the first section.
+

```

Comment:
Add line item for **Channel Id** as well. There are three Instance Details that user needs to enter.